docs/UserGuide.md

```diff
@@ -284,7 +286,18 @@ Format: `focus INDEX`
 * To switch to another Candidate's information, user will just need to type the new command and the new index of the candidate.
 
 ## Scheduling interviews
-
+**:information_source: Note:** 
+
+* The duration of an interview is fixed at 30 minutes. A new feature to allow the user to schedule interviews of flexible duration is
```

Comment:
Just wondering if it would be better to split up the note into "Upcoming Features" (as lines 316-317 are also introducing a future feature) and like maybe "Points to note" for ease of readability?

Reply:
Yup, I thought about it while writing it out too! Guess I should change it 😆